fsspec/implementations/dbfs.py

```diff
@@ -219,6 +226,60 @@ def _open(self, path, mode="rb", block_size="default", **kwargs):
         """
         return DatabricksFile(self, path, mode=mode, block_size=block_size, **kwargs)
 
+    def retry_request(func, retries=5):
```

Comment:
This is a function rather than a method, as it and `is_retiable` I think would be better as standalone functions.

Reply:
I have declared the retry_request as a class method because this decorator has been created solely for the DatabricksFilesystem class, if you think is better as a standalone function maybe we should create a new python file for the decorator and the retry.py on the gcsfs

Same with the is_retirable function